modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -225,6 +225,18 @@ TEST_P(Test_ONNX_layers, Gather)
     testONNXModels("gather", npy, 0, 0, false, false);
 }
 
+TEST_P(Test_ONNX_layers, RAFT)
+{
+    testONNXModels("RAFT", npy, 0, 0, false, true, 2);
```

Comment:
Change `required` to false before the test run. This is needed for large models tests to optionally skip it if there is no file on the CI:
```cpp
required = false;
testONNXModels("RAFT", npy, 0, 0, false, true, 2);
```

Reply:
Done